spaceship/lib/spaceship/connect_api/model.rb

```diff
@@ -25,7 +25,7 @@ def update_attributes(attributes)
       # Example:
       # { "minOsVersion" => "min_os_version" }
       #
-      # Creates attr_write and attr_reader for :min_os_version
+      # Creates attr_write and attr_/Users/tbodt/Developer/projects/fastlane/spaceship/spec/connect_api/models/model_spec.rb/Users/tbodt/Developer/projects/fastlane/spaceship/spec/connect_api/models/model_spec.rbreader for :min_os_version
```

Comment:
typo/leftover I guess?

Reply:
I need to figure out why my editor keeps randomly pasting :)